Vungle/VungleBannerCustomEvent.m

```diff
@@ -216,6 +215,12 @@ - (void)vungleAdDidAppear
     MPLogInfo(@"Vungle video banner did appear");
 }
 
+- (void)vungleAdViewed
+{
+    MPLogInfo(@"Vungle video banner did viewed");
```

Comment:
Grammar nitpick: Should be "did view" (or "was viewed"). Same with two instances below.

Reply:
@mikeh-vungle Hi Mike, thanks a lot for pointing out this issue. Fixed. :)